paper.md

```diff
@@ -151,40 +151,43 @@ bibliography: paper.bib
-The content representation is based on a checksum of the file's content.
-As such, while only a lightweight link is directly committed into Git, this committed information contains the content identity of annexed files.
-Git-annex then manages content availability information in any local repository, other Git remotes, or external resources such as web urls.
-Having that information, git-annex takes care of all transport logistics to exchange content upon user request.
-This simple approach allows git-annex to manage and version control arbitrarily large files, and "link" files in a Git repository to vast data resources available online.
-<!-- BEN thinks we need to make explicit the idea, that by means of how annex
-works, the content and access control to it is separated from the version
-control (and metadata). ATM this follows implicitly but may be much less obvious
-to someone who didn't dive into it yet -->
-
-## Why Git and git-annex alone are not enough
-
-<!-- this is "what does datalad add to Git+git-annex -->
+Code, data, and computing environments are at the core of scientific practice.
+Unobstructed access and efficient management of all those digital objects promotes scientific discovery through collaboration, reproducibility, and replicability.
+Established and widely-adopted infrastructure and procedures, like software distributions, distributed version control systems, and social coding portals like GitHub, streamline the collaborative development and use of research software.
+However, data have remained a “2nd-class citizen” in the contemporary scientific process, despite the increasing embracement of the FAIR principles [@FAIR2016].
+1) Disconnected data portals force a cacophony of data access and authentication methods upon the scientific community; 2) data versioning is rarely performed, or nowhere close to the precision and inspectability that is standard in software development; and 3) data provenance---if shared at all---is often not complete, because their capture is rarely considered to be an integral part of the scientific process.
+DataLad aims to solve all these issues by providing targeted interfaces and interoperability adapters to standard tools and services, scientific and commercial.
+It enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that affords automatic re-execution.
+In order to maximize its utility and target audience, DataLad is available for all major operating systems, and can be integrated into established workflows and environments with minimal adjustments.
+
+## Why Git and git-annex?
+
+Git is the standard version control system for software development[^1].
```

Comment:
there is no standard for designing version control systems, and "**the** standard" really throws me off as hinting on it to be the only one. What about simpler

```suggestion
Git is the most popular version control system for software development[^1].
```